Hearthstone Deck Tracker/Core.cs

```diff
@@ -85,7 +85,8 @@ public static class Core
 			var newUser = ConfigManager.PreviousVersion == null;
 			LogConfigUpdater.Run().Forget();
 			LogConfigWatcher.Start();
-			Helper.UpdateAppTheme();
+			UITheme.InitializeTheme();
+			UITheme.ShowHDTLogo();
```

Comment:
`ShowHDTLogo` doesn't seem to be named correctly. But that aside, why is this called separately here (a separate method at all) and not at the end of `InitializeTheme`?